tests/api_test.py

```diff
@@ -518,31 +515,6 @@ def test_template_generate_by_addr(self):
         assert gen_result.size() == 9
 
 
-class TestScKeynodes(ScTest):
-    class TestEnum(Enum):
-        IDTF_1 = "idtf_1"
-        IDTF_2 = "idtf_2"
-
-    def test_should_get_keynode(self):
-        self.get_server_message('{"errors": [], "id": 1, "event": false, "status": true, "payload": [1183238]}')
-        keynodes = ScKeynodes()
-        result = keynodes["SYS_IDTF"]
-        assert result.value == 1183238
-
-    def test_should_get_keynodes_by_enum(self):
-        self.get_server_message('{"errors": [], "id": 1, "event": false, "status": true, "payload": [1183238, 2]}')
-        keynodes = ScKeynodes()
-        keynodes.resolve_identifiers([self.TestEnum])
-        assert keynodes[self.TestEnum.IDTF_1.value].value == 1183238
-        assert keynodes[self.TestEnum.IDTF_2.value].value == 2
-
-    def test_should_get_unknown_idtf(self):
-        self.get_server_message('{"errors": [], "id": 1, "event": false, "status": true, "payload": [0]}')
-        keynodes = ScKeynodes()
-        result = keynodes["UNKNOWN_IDTF"]
-        assert not result.is_valid()
-
-
```

Comment:
Move this testcase to py-sc-kpm, just deleting is not good idea

Reply:
Added tests for keynodes
https://github.com/ostis-ai/py-sc-kpm/pull/25